pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1563,6 +1563,9 @@ type PoolPlacementSpec struct {
 	// +kubebuilder:validation:Pattern=`^[a-zA-Z0-9._/-]+$`
 	Name string `json:"name"`
 
+	// Sets given placement as default.
```

Comment:
For the generated docs, I think this note might be helpful.

```suggestion
	// Sets given placement as default. Only one placement in the list can be marked as default.
```

Reply:
done